lib/Twig/Extensions/Extension/HumanReadableBytes.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+use Twig_Error;
+use Twig_Extension;
+use Twig_SimpleFilter;
+
+class HumanReadableBytes extends Twig_Extension
```

Comment:
the class name must be ``Twig_Extensions_Extension_HumanReadableBytes`` to respect PSR-0 and be autoloadable. Otherwise the class is unusable